prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -280,7 +280,7 @@ ssize_t rxr_pkt_post_ctrl_once(struct rxr_ep *rxr_ep, int entry_type, void *x_en
 	if (inject)
 		err = rxr_pkt_entry_inject(rxr_ep, pkt_entry, addr);
 	else
-		err = rxr_pkt_entry_send(rxr_ep, pkt_entry, 0);
+		err = rxr_pkt_entry_send(rxr_ep, pkt_entry, flags);
```

Comment:
I think this should be part of previous commit, in which you introduced flags to the function.

Reply:
I made this as a separate commit because the previous one is a refactor, this one is the real fix. But I could combine them as suggested.

Reply:
Thanks! I think if you introduce an argument to a function, the argument should be used in the same commit.

Reply:
Fixed in latest revision.